### ISSUE TITLE
Issues with setting for creation of type stubs

### ISSUE THREAD

**Issue by MikhailArkhipov** [microsoft/pyright#494]
**Describe the bug**
Attempt to create type stub for external library produces confusing errors.

**To Reproduce**
- Create project with a single file like
```python
# pyright: reportMissingTypeStubs=true
import numpy
numpy.
```
- Observe code action `create type stub`. Click it.

OBSERVED: Error: `typingsPath` is not specified. 
Documentation says default would be `./typings`. From the code it looks like default is only set when configuration is initialized from JSON, i.e. `pyrightconfig.json` exists.

EXPECTED: default to work without having to create config file or message that says configuration file must be there.

- Create `pyrightconfig.json` at the root with
```json
{
    "typingsPath": "./typestubs"
}
```
- Reload window and try create stub again.

OBSERVED: 
Pyright creates `typestubs` folder and populates it with the stub. However, 
    - Output window says `typingsPath c:\Users\mikhaila\Documents\Test\PyTest1\typestubs is not a valid directory.`
    - Errors about missing stub are still there.

EXPECTED: no errors, folder created automatically and analysis restarted now that stub exists.

![image](https://user-images.githubusercontent.com/12820357/73411372-fe3cc580-42b9-11ea-8d17-208577755e95.png)

**Comment by erictraut** [comment]
This is fixed in Pyright 1.1.22, which I just published.
